bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/RowAdapter.java

```diff
@@ -54,17 +59,20 @@ public Result adaptResponse(Row response) {
             continue;
           }
 
-          long hbaseTimestamp =
-              BigtableConstants.HBASE_TIMEUNIT.convert(
-                  cell.getTimestampMicros(), BigtableConstants.BIGTABLE_TIMEUNIT);
-          KeyValue keyValue = new KeyValue(
-              rowKey,
-              familyNameBytes,
-              columnQualifier,
-              hbaseTimestamp,
-              cell.getValue().toByteArray());
-
-          hbaseCells.add(keyValue);
+          Long cellTimestamp = cell.getTimestampMicros();
+          if (!cellTimestamps.contains(cellTimestamp)) {
```

Comment:
What if there are different family/columns with the same cell timestamp?  I think this would only return one of those cells, when we should be returning 2.  We should also add a test for that condition for different family/column, same family/different column, different family/same column.